Add troubleshooting for apt-key and gpg (#2076)
ClickHouse/clickhouse-docs: docs/en/guides/troubleshooting.md

## Installation

### Cannot import GPG keys from keyserver.ubuntu.com with apt-key

> 1. `apt-key` is deprecated and it should use the `gpg` command instead. Please refer the [install guide](../getting-started/install.md) article.

Reviewer: ```suggestion
The `apt-key` feature with APT has been deprecated. Users should use the `gpg` command instead. Please refer the [install guide](../getting-started/install.md) article.
```

> ### Canot import GPG keys from keyserver.ubuntu.com with gpg

Reviewer: ```suggestion
### Cannot import GPG keys from `keyserver.ubuntu.com` with gpg
```

> 1. Check and upgrade the `gpg` version and it might be too outdated.

Reviewer: ```suggestion
1. See if there are any upgrades available for your `gpg` installation:

```shell
sudo apt update gpg
```

2. Ensuring the `dirmngr` package is installed.

> 3. If you cannot upgrade the `gpg` version, please use the following commands to import GPG keys:
> 
> ```shell
> GNUPGHOME=$(mktemp -d)
> sudo GNUPGHOME="$GNUPGHOME" gpg --no-default-keyring --keyring /usr/share/keyrings/clickhouse-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 8919F6BD2B48D754
> sudo rm -rf "$GNUPGHOME"
> sudo chmod +r /usr/share/keyrings/clickhouse-keyring.gpg
> ```

Reviewer: ```suggestion
```
